frontend/src/components/TaskValueCreatedVisualization.tsx

```diff
@@ -14,23 +14,37 @@ import { apiV2 } from '../api/api';
 
 const classes = classNames.bind(css);
 
+// either provide versions or customer stakes
 export const TaskValueCreatedVisualization: FC<{
-  versions: Version[];
+  stakes?: CustomerStakes[];
+  versions?: Version[];
   width: number;
   height: number;
   barWidth?: number;
   noTooltip?: true;
   vertical?: boolean;
-}> = ({ versions, width, height, noTooltip, vertical, barWidth = 37 }) => {
+}> = ({
+  stakes,
+  versions,
+  width,
+  height,
+  noTooltip,
+  vertical,
+  barWidth = 37,
+}) => {
```

Comment:
```suggestion
}> = ({ width, height, noTooltip, vertical, barWidth = 37, ...props }) => {
  const { stakes, versions } = {
    stakes: undefined,
    versions: undefined,
    ...props,
  };
```

The provided option can then be unpacked providing `undefined` for the other